pkg/kubelet/dockertools/docker.go

```diff
@@ -454,7 +454,7 @@ func GetDockerPodInfo(client DockerInterface, manifest api.PodSpec, podFullName
 	for _, container := range manifest.Containers {
 		expectedContainers[container.Name] = container
 	}
-	expectedContainers["net"] = api.Container{}
+	expectedContainers["POD"] = api.Container{}
```

Comment:
That constant is defined in the kubelet package. Is it okay to import kubelet here or should we define another constant here?